sidechain/consensus/aura/Cargo.toml

```diff
@@ -6,42 +6,64 @@ edition = "2018"
 
 [dependencies]
 log = { version = "0.4.14", default-features = false }
+finality-grandpa = { version = "0.14.3", default-features = false, features = ["derive-codec"] }
+sgx-externalities = { default-features = false, git = "https://github.com/integritee-network/sgx-runtime", branch = "master", optional = true }
 
 # sgx deps
 sgx_tstd = { branch = "v1.1.4-testing", git = "https://github.com/apache/teaclave-sgx-sdk.git", optional = true }
 
 # substrate deps
 frame-support = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master"}
+sp-core = { version = "4.0.0-dev", default-features = false, features = ["full_crypto"], git = "https://github.com/paritytech/substrate.git", branch = "master" }
 sp-runtime = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "master"}
 
 # local deps
+itp-settings = { path = "../../../core-primitives/settings", default-features = false }
+itp-sgx-crypto = { path = "../../../core-primitives/sgx/crypto", default-features = false }
+itp-stf-state-handler = { path = "../../../core-primitives/stf-state-handler", default-features = false }
 itp-storage-verifier = { path =  "../../../core-primitives/storage-verified", default-features = false }
-its-validateer-fetch = { path = "../../validateer-fetch", default-features = false }
+itp-time-utils = { path = "../../../core-primitives/time-utils", default-features = false }
+itp-types = { path = "../../../core-primitives/types", default-features = false }
+its-block-composer = { path = "../../block-composer", default-features = false }
 its-consensus-common = { path = "../common", default-features = false }
 its-consensus-slots = { path = "../slots", default-features = false }
 its-primitives = { path = "../../primitives", default-features = false }
 its-state = { path = "../../state", default-features = false }
+its-top-pool-executor = { path = "../../top-pool-executor", default-features = false }
+its-validateer-fetch = { path = "../../validateer-fetch", default-features = false }
```

Comment:
Because of the new components in this crate, a couple of new dependencies are required now.